package.json

```diff
@@ -140,15 +140,9 @@
     "@types/styled-components": "^5.1.4",
     "@types/twilio-video": "^2.7.2",
     "@types/yup": "^0.29.3",
-    "@typescript-eslint/eslint-plugin": "^3.7.1",
-    "@typescript-eslint/parser": "^3.7.1",
     "cypress": "^4.11.0",
     "cypress-file-upload": "^4.0.7",
-    "eslint-config-react-app": "^5.2.1",
-    "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-react": "^7.20.5",
-    "eslint-plugin-react-hooks": "^4.0.8",
```

Comment:
[note] These are all provided by `react-scripts`, so there is no need for us to specify them directly here like this (and doing so can lead to weird errors/version mismatches/etc)